### PR TITLE
ci: add assignees

### PR DESCRIPTION
**Pull request by xiaoyijun**
## Summary

ci: add assignees

## Linear Issue Reference

None.

## Testing

Run in CI

### CODE REVIEW

**Review by IceHe — COMMENTED** [review]
better merge `add-labels.yml` and `add-assignees.yml` into one file?

**Review by IceHe — CHANGES_REQUESTED** [review]
should the file name be `add-assignee.yml`?

the prefix `action-` and the plural form of `assignees` are both unnecessary.

File: .github/workflows/add-labels-and-assignees.yml [hunk 1]
```diff
@@ -14,3 +14,12 @@ jobs:
         with:
           title: ${{ github.event.pull_request.title || github.event.issue.title }}
           github-token: ${{ github.token }}
+
+  add_assignees:
```
Comment (IceHe) [inline .github/workflows/add-labels-and-assignees.yml:18]: better do them in one job and two steps?

**Comment by xiaoyijun** [comment]
Open a new pr to test if this works. #547

**Pull request closed without merging**